tests/onnx/quantization/test_quantizer_config.py

```diff
@@ -37,6 +43,9 @@ def check_is_min_max_statistic_collector(self, tensor_collector):
     def check_is_mean_min_max_statistic_collector(self, tensor_collector):
         assert isinstance(tensor_collector, ONNXMeanMinMaxStatisticCollector)
 
+    def get_reduction_axes(self, reducer: TensorStatisticCollectorBase) -> Tuple[int, ...]:
+        return reducer._reduction_shape
```

Comment:
But we don't need this in code, we need this only in tests

Reply:
The rule of thumb is that the private attributes should not be accessed even in tests. You are supposed to be testing external behaviour, not internal. You can "cheat" and make this behaviour external, or you can think about it as making your reducer more testable. Exposing a read-only property is pretty safe. Otherwise you need to rethink the test so that you are not testing the implementation details.

Reply:
We need to refactor whole nncf then. I think it's too much!!!

Reply:
Adding a property is not too much. Not proliferating bad practices in your own code at least, when you are pointed towards it explicitly, is not too much. Making the related code better even if it's not yours, one issue at a time, is not too much.

Reply:
I don't agree with you, let's discuss it offline

Reply:
As private attributes usage in tests is spreader across almost all tests in NNCF, I suggest to take a serious action and fix private attributes problem once for all tests. For this we need to add this action to the list for refactoring. What do you think?

Reply:
Ok